tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -2974,97 +2974,38 @@ Status ConvertRelu6(OpConverterParams* params) {
 Status ConvertBiasAdd(OpConverterParams* params) {
   const auto& inputs = params->inputs;
   const auto& node_def = params->node_def;
-  TF_RETURN_IF_ERROR(
-      CheckInputsWeights(*params, {{"value", false}, {"bias", true}}));
-  TF_RETURN_IF_ERROR(
-      AllowDataTypes(*params, {DataType::DT_FLOAT, DataType::DT_HALF}));
```

Comment:
Don't remove `CheckInputsWeights` or `AllowDataTypes`

Reply:
Wouldn't CheckInputsWeights fail if it expects a tensor/weight and the actual input is the opposite?

Reply:
Oh right, since the inputs will be able to be a tensor or weight now. So we should check that there are 2 inputs and that both inputs are not weights.